src/libexec/zmrestoreblob

```diff
@@ -14,21 +14,82 @@
 # If not, see <https://www.gnu.org/licenses/>.
 # ***** END LICENSE BLOCK *****
 
+=pod
+=head1 This utility is used to check orphan headers for given account, if founds any missing blobs, restore it
+
 #Usage
 # su - zimbra
-# zmrestoreblob -m 3
-# where -m is to specify which mailbox to check. In above case it is mailbox id 3
+# zmrestoreblob -a|account {name@domain|id}
+# zmrestoreblob -a test1@zcs-dev.test OR zmrestoreblob -a test1
+# where -a is to specify which mailbox account to check.
+
+=cut
 
 use strict;
 use warnings;    #or else.
 use lib "/opt/zimbra/common/lib/perl5";
 use Data::Dumper;
 use File::Basename;
 use File::Copy;
+use Archive::Zip qw( :ERROR_CODES :CONSTANTS );
+use Getopt::Long;
+
+
+my $opt_help;
+my $opt_account;
+GetOptions(
+    "help|h" => \$opt_help,
+    "account|a=s" => \$opt_account
+);
+
+if (defined($opt_help) || !defined($opt_account)) {
+    usage();
+}
+
+sub usage {
```

Comment:
don't write own usage function. use pod2usage.

Reply:
Also, please run the code through perltidy.

Reply:
Used pod2usage and run the code through perltidy - done